models.py

```diff
@@ -32,7 +32,7 @@ class Election(db.Model):
     organization_id = db.Column(
         db.String(200),
         db.ForeignKey("organization.id", ondelete="cascade"),
-        nullable=False,
+        nullable=True,
```

Comment:
I didn't really think it through when I first set this up. If we want to preserve anonymous creation of elections we either have to create an organization for each _or_ we allow the organization to be omitted. The latter makes it easier to control access since we no that "no organization" means "anonymous access allowed". Doing anything else would require more work, like a flag on the organization or something.